.npmignore

```diff
@@ -1 +0,0 @@
-test/
```

Comment:
this file should be kept to ignore files that will not be deployed to npm server.
like `test`, `examples`,`..travis.yml`

Reply:
the list should be in the `.npmignore` file.
- test/
- docs/
- examples
- .vscode/
- scripts/
- .travis.yml
- TODO

Reply:
actually the contents inside the dist folder will only be published as a package at npm

Reply:
got it.